utils/score.ts

```diff
@@ -0,0 +1,33 @@
+import {EntityOccurrence} from "~/utils/entities/occurrence";
```

Comment:
code style, here and rest of the file:
* `{EntityOccurrence}` -> ` {EntityOccurrence} `
* no semicolons